packages/core/echo/feed/src/sync.test.ts

```
@@ -3,101 +3,371 @@
 //
 
 import type * as SqlClient from '@effect/sql/SqlClient';
-import * as SqliteClient from '@effect/sql-sqlite-node/SqliteClient';
-import { describe, expect, it } from '@effect/vitest';
+import { describe, expect, test } from '@effect/vitest';
+import * as Array from 'effect/Array';
```

Biome (2.3.14)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)